person_class.rb

```diff
@@ -2,16 +2,18 @@
 require_relative './base_decorator'
 require_relative './trimer_decorator'
 require_relative './capitalize_decorate'
+require_relative './rental'
 
 class Person < Nameable
   attr_accessor :name, :age, :parent_permission
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Agin, `rental` should be marked with `attr_accessor`, as we intend to both _access (get)_ as well as _write (set)_ it. Kindly ensure to improve upon this aspect.

Reply:
This change has been done well